libtenzir/builtins/operators/chart2.cpp

```diff
@@ -752,18 +750,6 @@ class chart_operator2 final : public crtp_operator<chart_operator2> {
 
   auto validate_y(const data& d, std::string_view yname, tenzir::location loc,
                   diagnostic_handler& dh) const -> bool {
-    if (is<caf::none_t>(d)) {
-      if (args_.ty == chart_type::line) {
-        return true;
-      }
```

Comment:
Please add a changelog entry for this. Here's a suggestion:

> `chart_area`, `chart_bar`, and `chart_pie` no longer reject null-values. Previously, gaps in charts were only supported for `chart_line`.

The other change for always filling nulls when a resolution is given does not need a changelog entry, I think, as it doesn't actually change how we render the charts.